virtcontainers/device/drivers/vhost_user_blk.go

```diff
@@ -35,30 +43,75 @@ func (device *VhostUserBlkDevice) Attach(devReceiver api.DeviceReceiver) (err er
 	if skip {
 		return nil
 	}
+
+	// Since vhost-user-blk uses "vd" prefix in Linux kernel, not "sd",
+	// sandbox block index should be updated only if sandbox default block
+	// driver is "virtio-blk"/"virtio-blk-ccw"/"virtio-mmio" which uses
+	// "vd" prefix in Linux kernel.
+	updateBlockIndex := false
+	customOptions := device.DeviceInfo.DriverOptions
+	if customOptions["block-driver"] == "virtio-blk" ||
+		customOptions["block-driver"] == "virtio-blk-ccw" ||
+		customOptions["block-driver"] == "virtio-mmio" {
```

Comment:
how about saving `customOptions["block-driver"]` in a variable?

Reply:
ack